string name
uint32 index
uint32 value
---
uint32 ret